components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementServiceImpl.java

```diff
@@ -383,6 +383,14 @@ public void addWorkflow(Workflow workflow,
 
     }
 
+    @Override
+    public String getTemplateId(String workflowId) throws WorkflowException {
+        return WorkflowRequestDAO.retrieveTemplateId(workflowId);
```

Comment:
When the static modified of the methods are removed, the methods will have to be accessed as
`workflowDAO.retrieveTemplateId(workflowId);`